cernopendata/modules/fixtures/data/docs/cod-privacy-policy/cod-privacy-policy.md

```diff
@@ -1 +1,61 @@
-CERN does not track, collect or retain personal information from users of CERN Open Data Portal, except as otherwise provided herein. In order to enhance CERN Open Data Portal and monitor traffic, information such as IP addresses and cookies may be tracked and retained, as well as log files shared in aggregation with other community services. User provided information, like corrections of metadata or paper claims, will be integrated into the database without displaying its source and may shared with other services. CERN Open Data Portal will take all reasonable measures to protect the privacy of its users and to resist service interruptions, intentional attacks, or other events that may compromise the security of the CERN Open Data Portal website. If you have any questions about the CERN Open Data Portal privacy policy, please contact [opendata-support@cern.ch](mailto:opendata-support@cern.ch)
+CERN does not track, collect or retain personal information from users of CERN Open Data Portal, except as otherwise provided herein. In order to enhance CERN Open Data Portal and monitor traffic, information such as IP addresses and cookies may be tracked and retained, as well as log files shared in aggregation with other community services. User provided information, like corrections of metadata or paper claims, will be integrated into the database without displaying its source and may shared with other services.
+
+CERN Open Data Portal will take all reasonable measures to protect the privacy of its users and to resist service interruptions, intentional attacks, or other events that may compromise the security of the CERN Open Data Portal website.
+
+If you have any questions about the CERN Open Data Portal privacy policy, please contact [opendata-support@cern.ch](mailto:opendata-support@cern.ch)
+
+
+## How is your data used
+
+Each service at CERN is responsible for compiling its own Privacy Notice regarding the data it processes.
+
+This Privacy Notice is part of [CERN’s Layered Privacy Notice](https://cern.service-now.com/service-portal?id=layered_privacy_notice) and details the processing that is unique to CERN Open Data Portal. It does not address processing by other services on which this service may rely and which have their own Privacy Notice.
+
+### Personal Data we process
+
+The personal data we have, and how it's used:
+
+Personal Data | Purpose | Basis | Source
```

Comment:
Haven't checked the generated code, but  perhaps injecting HTML here instead of using Markdown would be better so that  tables are nicely bordered?